src/Features/CSharp/Portable/ReplacePropertyWithMethods/CSharpReplacePropertyWithMethodsService.cs

```diff
@@ -132,8 +132,13 @@ public override SyntaxNode GetPropertyDeclaration(SyntaxToken token)
 
             if (propertyDeclaration.ExpressionBody != null)
             {
-                var returnStatement = SyntaxFactory.ReturnStatement(propertyDeclaration.ExpressionBody.Expression)
-                                                   .WithSemicolonToken(propertyDeclaration.SemicolonToken);
+                var returnKeyword = SyntaxFactory.Token(SyntaxKind.ReturnKeyword)
+                                                 .WithTrailingTrivia(propertyDeclaration.ExpressionBody.ArrowToken.TrailingTrivia);
+
+                var returnStatement = SyntaxFactory.ReturnStatement(
+                    returnKeyword, 
+                    propertyDeclaration.ExpressionBody.Expression,
+                    propertyDeclaration.SemicolonToken);
```

Comment:
I see we're taking the trivia from `ArrowToken`. What about trivia from the expression itself, is it retained? Case: `=> 42; /* return 42 */`

If that works, could you add a test?